pkg/store/bucket.go

```diff
@@ -309,6 +319,17 @@ func (s *BucketStore) SyncBlocks(ctx context.Context) error {
 		if err != nil {
 			return nil
 		}
+
+		inRange, err := s.isBlockInMinMaxRange(ctx, id)
```

Comment:
I think the problem here is that we hit the issue of partial blocks... Something that https://github.com/thanos-io/thanos/pull/1394 touches. 

I thought we actually handle this in store gw properly (blocks without meta are excluded), seems not so not a regression.

In any case we will just warn so :+1: